gulpfile.js

```diff
@@ -70,4 +74,21 @@ gulp.task('vendor-script', async () => {
         fs.mkdirSync(targetFolder);
     }
     await writeFileAsync(path.join(targetFolder, 'vendor.js'), result.code);
+});
+
+gulp.task('design-file', async () => {
+    let stylesdir = path.join(__dirname, './components/styles')
+    let content = '';
+
+    for (let file of fs.readdirSync(stylesdir) ) {
+        if (file.startsWith('_') ) {
```

Comment:
I'm also not sure if we should check that '_' at the beginning of the files, probably just check the extensions?

Reply:
I agree with also checking on file extension, but we do need to check on the underscore (or just remove that notation because why is it even needed?) since we need to strip that underscore from the filename if we want to use it as input for the design.scss file imports (so we have to expect that it is there).

Is that underscore even necessary (also @wwsno)?

Reply:
Now it (underscore) is expected by the client, so it is necessary for now. Lets keep that checking as well.